src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexPrefixAnalyzer.cs

```diff
@@ -865,6 +896,19 @@ private static RegexNodeKind FindLeadingOrTrailingAnchor(RegexNode node, bool le
             }
         }
 
+        private static float SumFrequencies(ReadOnlySpan<char> chars)
+        {
+            float sum = 0;
+            foreach (char c in chars)
+            {
+                // Lookup each character in the table.  For values > 255, this will end up truncating
+                // and thus we'll get skew in the data.  It's already a gross approximation, though,
+                // and it is primarily meant for disambiguation of ASCII letters.
+                sum += s_frequency[(byte)c];
```

Comment:
s_frequency excludes newline characters (ReadLine) - are there any circumstances where frequencies of those might be useful?